app/assets/stylesheets/properties.css

```
@@ -346,24 +346,71 @@
   -webkit-backdrop-filter: none !important;
   transform: none;
   transition: transform 0.3s ease, border-color 0.3s ease, box-shadow 0.3s ease;
+  image-rendering: crisp-edges;
+  image-rendering: -webkit-optimize-contrast;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

image-rendering is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)